### PR TITLE
Profile Backend

### PR DESCRIPTION
**Pull request by alexapostolu**
The front end now requests user information from the database!

**Note:** For the user routes, it uses the username like so, `api/users/username`

### CODE REVIEW

File: client/src/pages/Profile.tsx [hunk 1]
```diff
@@ -10,10 +10,10 @@ const Profile = () => {
   const [comments, setComments] = useState([]);
 
   useEffect(() => {
-    fetch('/api/users/1')
+    fetch('/api/users/Apostolu')
```
Comment (arthurhohinng) [inline client/src/pages/Profile.tsx:13]: This is to test that it works but we want to merge the logic that will work for any user

**Review by arthurhohinng — COMMENTED** [review]
This is to test that it works right?
And since you said it works we can write the logic to make it work for any users

So instead of `api/users/Apostolu` it should take in the utorid and then fetch the userinfo from that
Or if we are fetching using user id the user id is passed as a parameter

But we want it to be able to fetch users other than this test user

File: client/src/pages/Profile.tsx (hunk 1 above)
Comment (alexapostolu) [reply client/src/pages/Profile.tsx:13]: I thought we were waiting for Shibboleth to get set up first. Because I am not sure how to get the utorid

Comment (Vladmidir) [reply client/src/pages/Profile.tsx:13]: You should really get the profile utorid from the url params. Lookup the `useParams()` hook.

File: server/scripts/importUsers.js
Comment (arthurhohinng) [inline server/scripts/importUsers.js:1]: We don't need to commit this file since it was just used to test on your end locally

**Review by arthurhohinng — APPROVED** [review]
If tested and functional looks good 
You should remove import users before merging though

**Pull request merged**